sdks/python/apache_beam/runners/interactive/utils.py

```diff
@@ -205,7 +207,9 @@ def __enter__(self):
       else:
         display(self._enter_text)
     except ImportError:
-      pass  # NOOP when dependencies are not available.
+      _LOGGER.warning(
```

Comment:
It'd be helpful to log the ImportError here as well.

Reply:
Added the log of ImportError.